apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/config/BizConfig.java

```diff
@@ -99,6 +102,15 @@ public int itemValueLengthLimit() {
     return checkInt(limit, 5, Integer.MAX_VALUE, DEFAULT_ITEM_VALUE_LENGTH);
   }
 
+  public boolean isItemNumLimitEnabled() {
+    return getBooleanProperty("item.num.limit.enabled", false);
+  }
+
+  public int itemNumLimit() {
+    int limit = getIntProperty("item.num.limit", DEFAULT_MAX_ITEM_NUM);
+    return checkInt(limit, 5, Integer.MAX_VALUE, DEFAULT_MAX_ITEM_NUM);
```

Comment:
Why is the minimum 5?

Reply:
The minimum value of 5 is set based on the values ​​of `item.key.length.limit `and `item.value.length.limit`. Considering that actual use will not set a value smaller than 5.
Can you give me a suggested minimum?